src/store/traction/saphyr/tubes/actions.js

```diff
@@ -71,19 +71,37 @@ const deleteLibraries = async ({ getters }, libraryIds) => {
   return responses
 }
 
+const setLibraries = async ({ commit, getters }) => {
+  let request = getters.libraryRequest
+  let promise = request.get()
+  let response = await handlePromise(promise)
+  let libraries = null
+
+  if (response.successful && !response.empty) {
+    libraries = response.deserialize.libraries
+    
+    commit('setLibraries', libraries)
+  }
+
+  return libraries
```

Comment:
may not need to return anything from this function

Reply:
I am using the results from the action in the test. Don't know if there is a better way?